src/mscorlib/src/System/Span.cs

```diff
@@ -289,6 +289,25 @@ public void Set(ReadOnlySpan<T> values)
 
             SpanHelper.CopyTo<T>(ref GetRawPointer(), ref values.GetRawPointer(), values.Length);
         }
+
+        /// <summary>
+        /// Fills the contents of this span with the value.
+        /// </summary>
+        /// <param name="value">The value to fill the span content with.</param>
+        public void Fill(T value)
+        {
+            if (_length > 0)
+                SpanHelper.Fill<T>(ref GetRawPointer(), value, _length);
+        }
+
+        /// <summary>
+        /// Fills the contents of this span with the default value.
+        /// </summary>
+        public void Clear()
+        {
+            if (_length > 0)
+                SpanHelper.Fill<T>(ref GetRawPointer(), default(T), _length);
```

Comment:
If I understand correct. If type T `ContainsReferences` then we can not just clear all array items to 0. We must do item by item clear. Am I right?